cg/cli/clean.py

```diff
@@ -304,3 +305,19 @@ def _get_confirm_question(bundle, file_obj) -> str:
         if file_obj.is_included
         else f"{bundle}: remove file from database: {file_obj.full_path}"
     )
+
+
+@clean.command("analysis-cases")
+@click.option(
+    "--days-old",
+    type=int,
+    default=60,
+    help="Clean the <environment-dir>/analysis/cases directory of old files.",
+    show_default=True,
+)
+@DRY_RUN
+@click.pass_obj
+def clean_analysis_cases(context: CGConfig, days_old: int, dry_run: bool):
+    """Clean the ~/analysis/cases directory of old files"""
+    directory_to_clean = Path(context.tomte.root)
```

Comment:
Maybe this could be moved to the servers. There are plans to move the common tower information to a new section. This could be part of it